tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Tests/AdaptiveDialogBotTests.cs

```diff
@@ -56,6 +56,8 @@ public async Task AdaptiveDialogBotTurnState()
 
             var turnContext = new TurnContext(adapterMock.Object, activity);
 
+            var telemetryClient = new NullBotTelemetryClient();
```

Comment:
Can we see a test where if instanceId is null [ActionScope:323](https://github.com/microsoft/botbuilder-dotnet/pull/5509/files#diff-0d6a44d510dea34dfbc9c2006ec0ebeaadada3a054dd739e37403888b82c9e5eR323), nothing breaks? This is my main concern with this PR

Reply:
If we test that we'd be safe.

Reply:
Removed.